test/tools/WebListener/Controllers/DosController.cs

```diff
@@ -29,33 +29,37 @@ public string Index()
             }
 
             StringValues dosLengths;
-            Int32 dosLength =1;
+            Int32 dosLength = 1;
             if (Request.Query.TryGetValue("dosLength", out dosLengths))
             {
                 Int32.TryParse(dosLengths.FirstOrDefault(), out dosLength);
             }
 
             string body = string.Empty;
-            switch(dosType)
+            switch (dosType)
             {
                 case "img":
                     contentType = "text/html; charset=utf8";
                     body = "<img" + (new string(' ', dosLength));
                     break;
+                case "img-attribute":
```

Comment:
This isn't actually a DOS... can you just add a comment explaining that?

Reply:
@TravisEz13 Yeah I was torn on that myself. I could also just add a new Controller class here instead, since we plan on adding future cases to test the actual regex patterns' capability to parse what we expect?

Reply:
I don't think you need to.  I'll leave it up to you, but at least add a comment that this is not a DOS case, but you didn't want to create a new controller for a case that was so similar.